server/sandbox_run_linux.go

```diff
@@ -385,6 +388,28 @@ func (s *Server) runPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 	}
 	g.AddAnnotation(annotations.CgroupParent, cgroupParent)
 
+	// check memory limit is greater than the minimum memory limit of 4Mb
```

Comment:
Can we move this up under the `if s.config.CgroupManager == oci.SystemdCgroupsManager {` code block?
This won't work for cgroupfs.

Reply:
right, my bad. Fixed